nixpkgs/toolchains/cc.nix

```diff
@@ -176,10 +179,11 @@ in
           # Have gcc return the exit code from ld.
           add_compiler_option_if_supported -pass-exit-codes
         )
+      )
+      LINK_LIBS=(
```

Comment:
Just for reference, this change is in line with `--incompatible_linkopts_to_linklibs` which has [recently been flipped](https://github.com/bazelbuild/bazel/issues/10905) to be enabled by default.